pinot-core/src/main/java/com/linkedin/pinot/core/segment/index/ColumnMetadata.java

```diff
@@ -50,8 +50,11 @@
   private final int stringColumnMaxLength;
   private final FieldType fieldType;
   private final boolean isSorted;
+  @JsonProperty
```

Comment:
Because jackson handles these booleans differently and does not output by default